modules/visual_script/visual_script_editor.h

```diff
@@ -165,23 +170,27 @@ class VisualScriptEditor : public ScriptEditorBase {
 	int port_action_output;
 	Vector2 port_action_pos;
 	int port_action_new_node;
-	void _port_action_menu(int p_option);
 
-	void new_node(Ref<VisualScriptNode> vnode, Vector2 ofs);
+	bool saved_pos_dirty;
+	Vector2 saved_position;
+
+	Vector2 mouse_up_position;
+
+	void _port_action_menu(int p_option, const StringName &func);
```

Comment:
Parameters should be prefixed with `p_`, so this should be `p_func`. You should review most changed or added prototypes, quite a few don't follow this convention.

Reply:
Will do.